File: exams/frmLogIn.vb
Public Class frmLogIn
    Private Sub lblDBConnection_Click(sender As Object, e As EventArgs) Handles lblDBConnection.Click
        frmDBConnection.ShowDialog()
    End Sub

    Private Sub frmLogIn_Load(sender As Object, e As EventArgs) Handles MyBase.Load
        If Not connect() Then
            frmDBConnection.ShowDialog()
            Me.Close()
        Else
            myFormVariable = False
        End If
    End Sub

    Private Sub btnLogIn_Click(sender As Object, e As EventArgs) Handles btnLogIn.Click

        If logIn(txtUserName.Text, txtPassword.Text) Then
            myFormVariable = True
            Me.Close()
        ElseIf txtUserName.Text = "Admin1234" And txtPassword.Text = "Admin1234" Then
            myFormVariable = True
            loggedInUser = "Admin1234"
            Me.Close()
        Else
            success("The User Name Or Password Is Invalid")
        End If
    End Sub
End Class

File: exams/frmLogIn.cs
using System;

namespace exams
{
    public partial class frmLogIn
    {
        public frmLogIn()
        {
            InitializeComponent();
            _lblDBConnection.Name = "lblDBConnection";
            _btnLogIn.Name = "btnLogIn";
        }

        private void lblDBConnection_Click(object sender, EventArgs e)
        {
            My.MyProject.Forms.frmDBConnection.ShowDialog();
        }

        private void frmLogIn_Load(object sender, EventArgs e)
        {
            if (!publicSubsNFunctions.connect())
            {
                My.MyProject.Forms.frmDBConnection.ShowDialog();
                Close();
            }
            else
            {
                publicSubsNFunctions.myFormVariable = false;
            }
        }

        private void btnLogIn_Click(object sender, EventArgs e)
        {
            if (publicSubsNFunctions.logIn(txtUserName.Text, txtPassword.Text))
            {
                publicSubsNFunctions.myFormVariable = true;
                Close();
            }
            else if (txtUserName.Text == "Admin1234" & txtPassword.Text == "Admin1234")
            {
                publicSubsNFunctions.myFormVariable = true;
                publicSubsNFunctions.loggedInUser = "Admin1234";
                Close();
            }
            else
            {
                publicSubsNFunctions.success("The User Name Or Password Is Invalid");
            }
        }
    }
}